### PR TITLE
Stableswap Masterchef refactor

### PR DESCRIPTION
**Pull request by t0mcr8se**

### CODE REVIEW

File: src/utils/fetchChefPairInfo.ts [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+  let result
+  if (isStableswap(address)) {
+    result = await stableswapClient.query({
+      query: stablePoolQuery(address)
+    })
+    const tokens = result?.data?.swaps?.tokens?.map((token) => token?.id)
+    const reserves = result?.data?.swaps?.balances
+    const pooledTokensUsd = result?.data?.swaps?.balanes?.map(async (balance, i) => (await fetchTokenPrice(tokens[i], NetworkId.FUSE)) * balance)
+    return {
+      reserveUSD: pooledTokensUsd?.reduce((mem, balanceUsd, i) => mem + weiToNumber(balanceUsd, result?.data?.swaps?.tokens?.[i]?.decimals)),
```
Comment (t0mcr8se) [inline src/utils/fetchChefPairInfo.ts:17]: @mul53 use virtualPrice instead?

File: src/constants/index.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ export const UNISWAP_SUBGRAPH_URL = 'https://api.thegraph.com/subgraphs/name/uni
 export const FUSESWAP_SUBGRAPH_URL = 'https://api.thegraph.com/subgraphs/name/fuseio/fuseswap'
 export const VOLTAGE_SUBGRAPH_URL = 'https://api.thegraph.com/subgraphs/name/voltfinance/voltage-exchange'
 export const PANCAKESWAP_SUBGRAPH_URL = 'https://bsc.streamingfast.io/subgraphs/name/pancakeswap/exchange-v2'
+export const STABLESWAP_SUBGRAPH_URL = 'https://api.thegraph.com/subgraphs/name/t0mcr8se/stableswap-subgraph'
```
Comment (t0mcr8se) [inline src/constants/index.ts:11]: Need to change to voltfinance subgraph

File: src/utils/fetchChefPairInfo.ts (hunk 1 above)
Comment (mul53) [reply src/utils/fetchChefPairInfo.ts:17]: we can use `lpTokenSupply * 1` for the tvl

File: src/constants/index.ts (hunk 2 above)
Comment (mul53) [reply src/constants/index.ts:11]: https://thegraph.com/hosted-service/subgraph/voltfinance/stableswap

**Review by mul53 — APPROVED** [review]

**Pull request merged**